Applications/PlazaDetails/Strength.qmd

```diff
@@ -1,27 +1,32 @@
 ---
-title: "Credit Strength"
+title: "Credit Consumption"
 aliases: 
   - Application/PlazaDetails/Strength.html
 ---
 
-The following mentioned are among the popular services available in openEO algorithm plaza. Here we have presented an average credit strength for these services. Please note that depending on the time interval and area of interest, the credits can slightly vary.
+The following are among the popular services available in OpenEO Algorithm Plaza. Here, we have presented an average credit strength for these services. Please note that the credits can vary slightly depending on the time interval and area of interest.
 
 The purpose of this document is to solely provide users with an assumption on how these credits work and their strength for varying services.
 
 ##### NDVI
-For the calculation of Normalized Difference Vegetation Index (NDVI), you can acquire ~250 NDVI images, each with an area of 1 hectare, i.e. 1 NDVI image patch of 1 hectare cost 3-5 credits.
+
+The Normalized Difference Vegetation Index (NDVI) is a key indicator of vegetation health. It is calculated by comparing the difference between near-infrared (which vegetation strongly reflects) and red light (which vegetation absorbs) in satellite imagery.
+Calculating the NDVI of a square hectare costs 3-5 credits per hectare. 
 
 ##### NDII
-Computation of the Normalized Difference Infrared Index (NDII) provides the user with an indication of the water content in the plant canopies. A straightforward calculation of the NDII can be achieved from openEO algorithm plaza NDII service with the use of 4-8 credits per hectare area, i.e. you can use your 1000 credits for computing NDII in a given area of ~117 hectares.
+
+The Normalized Difference Infrared Index (NDII) computation offers insight into plant canopies' water content. Using the openEO Algorithm Plaza, NDII service costs 4-8 credits per hectare.
```

Comment:
I think the cost sentence here is cleaner, might want to copy ti over for the examples below